src/App.tsx

```diff
@@ -1,61 +1,111 @@
+import React, { useState } from 'react';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [addedToDos, setAddedToDos] = useState([...todosFromServer]);
+  const [titleError, setTitleError] = useState(false);
+  const [userError, setUserError] = useState(false);
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setTitleError(false);
+  };
+
+  const clear = () => {
+    setTitle('');
+    setUserId(0);
+  };
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (userId === 0) {
+      setUserError(true);
+    } else {
+      setUserError(false);
+    }
+
+    if (title === '') {
+      setTitleError(true);
+    } else {
+      setTitleError(false);
+    }
```

Comment:
```suggestion
      setTitleError(!title);
```

can be simplify also